src/test/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/vcf/VcfFuncotationFactoryUnitTest.java

```diff
@@ -286,4 +291,159 @@ public void testCreateFuncotationMetadata(final VariantContext variant,
+                        Arrays.asList(ImmutableMap.of("_AC_AMR", "0", "_AC_Het", "1,0,0", "_AC", "7","_DP_HIST", "428|427|186|183|1953|705|127|19|1|2|1|0|0|0|0|0|0|0|0|0,0|0|1|0|1|1|0|1|0|0|0|0|0|0|0|0|0|0|0|0"),
+                                ImmutableMap.of("_AC_AMR", "0", "_AC_Het", "1,0,0", "_AC", "2","_DP_HIST", "428|427|186|183|1953|705|127|19|1|2|1|0|0|0|0|0|0|0|0|0,0|1|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0"))},
+                //HARD!!  Same as the previous test
+                {new SimpleInterval("3", 324682, 324682), Arrays.asList("A", "T"),
+                        Collections.singletonList(ImmutableMap.of("_AC_AMR", "0", "_AC_Het", "1,0,0", "_AC", "7","_DP_HIST", "428|427|186|183|1953|705|127|19|1|2|1|0|0|0|0|0|0|0|0|0,0|0|1|0|1|1|0|1|0|0|0|0|0|0|0|0|0|0|0|0"))},
+
+                // Control case (no multiallelics)
+                //3	13372	.	G	C AC=3; AC_AMR=0 AC_Het=0 DP_HIST=14728|2455|2120|518|121|499|534|314|111|21|10|2|2|0|0|0|0|0|0|0,1|0|0|0|1|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0
+                {new SimpleInterval("3", 13372, 13372), Arrays.asList("G", "C"),
+                        Collections.singletonList(ImmutableMap.of("_AC_AMR", "0", "_AC_Het", "0", "_AC", "3","_DP_HIST", "14728|2455|2120|518|121|499|534|314|111|21|10|2|2|0|0|0|0|0|0|0,1|0|0|0|1|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0"))},
+
+                // Control case (no multiallelics in datasource, but multiallelic query) -- Should produce a funcotation for both alleles, but second one should be blank
+                //3	13372	.	G	C AC=3; AC_AMR=0 AC_Het=0 DP_HIST=14728|2455|2120|518|121|499|534|314|111|21|10|2|2|0|0|0|0|0|0|0,1|0|0|0|1|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0
+                {new SimpleInterval("3", 13372, 13372), Arrays.asList("G", "C", "T"),
+                        Arrays.asList(ImmutableMap.of("_AC_AMR", "0", "_AC_Het", "0", "_AC", "3","_DP_HIST", "14728|2455|2120|518|121|499|534|314|111|21|10|2|2|0|0|0|0|0|0|0,1|0|0|0|1|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0"),
+                                ImmutableMap.of("_AC_AMR", "", "_AC_Het", "", "_AC", "","_DP_HIST", ""))},
+                // Control case (no multiallelics in datasource, but multiallelic query) -- Should produce a funcotation for both alleles, but first one should be blank
+                //3	13372	.	G	C AC=3; AC_AMR=0 AC_Het=0 DP_HIST=14728|2455|2120|518|121|499|534|314|111|21|10|2|2|0|0|0|0|0|0|0,1|0|0|0|1|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0
+                {new SimpleInterval("3", 13372, 13372), Arrays.asList("G", "T", "C"),
+                        Arrays.asList(ImmutableMap.of("_AC_AMR", "", "_AC_Het", "", "_AC", "","_DP_HIST", ""),
+                                ImmutableMap.of("_AC_AMR", "0", "_AC_Het", "0", "_AC", "3","_DP_HIST", "14728|2455|2120|518|121|499|534|314|111|21|10|2|2|0|0|0|0|0|0|0,1|0|0|0|1|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0"))},
+        };
+    }
+
+    /**
+     * This also tests the VCFFuncotationFactory Caching.
+     */
+    @Test(dataProvider = "provideMultiallelicTest")
```

Comment:
Can you modify this test to also run the 3 argument version of `createFuncotationsOnVariant`?  I want to keep all entry points tested.

Reply:
See next comment.

No action.

Reply:
(More like "Done")